hash/archimedes.xml

```diff
@@ -197,155 +197,181 @@ Zalaga by ProAction (RiscPC & StrongARM)
 Zelanites: The Onslaught by Micro Power (RiscOS2)
 -->
 
-	<!-- boot OK, !Control throws a "sprite doesn't exist" exception -->
 	<software name="2067bc" supported="partial">
 		<description>2067 BC</description>
 		<year>1992</year>
 		<publisher>Oregan Software</publisher>
+		<notes><![CDATA[
+boot OK, !Control throws a "sprite doesn't exist" exception
+]]></notes>
 		<part name="flop1" interface="floppy_3_5">
 			<dataarea name="flop" size="819200">
 				<rom name="2067bc.adf" size="819200" crc="11383a74" sha1="af1441ef677fb02a864f709dd2c74e154f963511" />
 			</dataarea>
 		</part>
 	</software>
 
-	<!-- boot OK, mouse is very imprecise -->
 	<software name="3dconkit" supported="partial">
 		<description>3D Construction Kit</description>
 		<year>1991</year>
 		<publisher>Domark</publisher>
+		<notes><![CDATA[
+boot OK, mouse is very imprecise
```

Comment:
Notes are quite old format (to not say outdated), also I ditched the idea of using `boot OK` since it's not all that useful.

Reply:
I didn't change or review the content of the comments. So I believe some of them aren't useful.

> Notes are quite old format (to not say outdated)
I don't understand what you mean. The `notes` tag format is outdated?

Reply:
No, the notes _contents_ of this specific file aren't updated, given they predates when @Pernod70 start doing work on it (5+ years ago). Since then much has changed, for example the HDD games needs to be checked again.

Reply:
ok. maybe now it can be easier to check and update it.

Reply:
Definitely, thanks for converting :)